tethysext/atcore/controllers/resources/tabs/resource_list_tab.py

```diff
@@ -0,0 +1,76 @@
+"""
+********************************************************************************
+* Name: resource_list_tab.py
+* Author: gagelarsen
+* Created On: December 11, 2020
+* Copyright: (c) Aquaveo 2020
+********************************************************************************
+"""
+from django.shortcuts import reverse
+
+from .resource_tab import ResourceTab
+
+
+class ResourceListTab(ResourceTab):
+    """
+    # A tab for the TabbedResourceDetails view that lists resources related to this resource.
+
+    Required URL Variables:
+        resource_id (str): the ID of the Resource.
+        tab_slug (str): Portion of URL that denotes which tab is active.
+
+    Methods:
+        get_resources (required): Get a list of resources to be associated with this resource.
+    """  # noqa: E501
+    template_name = 'atcore/resources/tabs/resource_list_tab.html'
+    base_template = None
+    post_load_callback = 'resource_list_tab_loaded'
+
+    js_requirements = ResourceTab.js_requirements + [
+        'atcore/resources/resource_list_tab.js',
+        # TODO: Remove if we don't use this.
+    ]
+    css_requirements = ResourceTab.css_requirements + [
+        'atcore/resources/resource_list_tab.css'
+        # TODO: Remove if we don't use this.
+    ]
+
+    def get_resources(self, request, resource, session, *args, **kwargs):
+        """
+        Get a list of resources
+
+        Returns:
+            A list of FileCollection clients.
+        """
+        return []
+
+    def _build_resource_cards(self, resources):
+        resource_cards = []
+        for resource in resources:
+            resource_card = resource.__dict__
+            resource_card['debugging'] = resource.attributes
+            resource_card['debugging']['id'] = str(resource.id)
```

Comment:
I don't think this debugging information is used is it? Remove if not.

Reply:
I think you are right. removed.